README.md

```diff
@@ -92,47 +92,97 @@ The milestone needs testing on all platforms. Please report issues when found.
     - D3 Open Source compiles level scripts in their own hogfiles. Make sure you copy and overwrite `d3-{platform}.hog`.
 
 ## Building
-Build steps below assume you have already cloned the repository and entered it locally.
-
-#### Building - Windows
-Requires Visual Studio 2022 and C++ Tools (cmake and vcpkg)
-
-Install and configure vcpkg:
-```sh
-git clone https://github.com/microsoft/vcpkg.git
-cd vcpkg && bootstrap-vcpkg.bat
-setx VCPKG_ROOT="C:\path\to\vcpkg"
-setx PATH=%VCPKG_ROOT%;%PATH%
-```
-
-Build Descent 3:
-```sh
-cmake --preset win -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset win --config [Debug|Release]
-```
-
-#### Building - MacOS
-```sh
-brew bundle install
-cmake --preset mac -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset mac --config [Debug|Release]
-```
-
-#### Building - Linux (Ubuntu)
-```sh
-sudo apt update
-sudo apt install -y --no-install-recommends ninja-build cmake g++ libsdl2-dev zlib1g-dev
-cmake --preset linux -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset linux --config [Debug|Release]
-```
-
-#### Building - Linux (Fedora)
-```sh
-sudo dnf update --refresh
-sudo dnf install -y ninja-build cmake gcc-c++ SDL2-devel zlib-devel
-cmake --preset linux -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset linux --config [Debug|Release]
-```
+1. Make sure that any prerequisites are installed:
+
+    <details>
+    <summary><b>Windows</b></summary>
+
+    You’ll need Git and Visual Studio 2022 with the “Desktop development with C++” workload. If you don’t already have those installed or you aren’t sure, then open an elevated Command Prompt and run:
+
+    <!--
+    The following code block specifies the full path to the Visual Studio Installer because the Visual Studio Installer doesn’t add itself to the user’s Path. The installer is guaranteed to be in a specific location on 64-bit systems [1]. The installer will be in a different location on 32-bit systems [2], but Visual Studio 2022 doesn’t support 32-bit systems [3] so we can ignore that detail.
+
+    [1]: <https://learn.microsoft.com/en-us/visualstudio/install/use-command-line-parameters-to-install-visual-studio?view=vs-2022>
+    [2]: <https://github.com/microsoft/vswhere/wiki#installing>
+    [3]: <https://learn.microsoft.com/en-us/answers/questions/1689898/does-visual-studio-build-tools-2022-support-32-bit>
+    -->
+
+    ```batch
+    winget install Git.Git Microsoft.VisualStudio.2022.Community
+
+    "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\setup.exe" modify^
+        --passive^
+        --channelId VisualStudio.17.Release^
```

Comment:
A simple link like [this one](https://visualstudio.microsoft.com/downloads/) maybe be easier for us to keep up to date than a command to download a specific version of VS, what's your opinion @JeodC ?

Reply:
That's how I installed VS anyway. I'm not at all sure what the above is supposed to do. Being user friendly > being fast, since what's faster for one person may well be slower for another. User friendliness guarantees a similar process speed for all.

Reply:
> A simple link like [this one](https://visualstudio.microsoft.com/downloads/) maybe be easier for us to keep up to date than a command to download a specific version of VS,

I think that it would technically be easier to keep a link up to date, but I don’t think that it would be significantly easier. Once Descent 3 switches to a newer version of Visual Studio, we would only have to change two numbers. We would have to change the 2022 to whatever the new year is, and we would have to change the 17 to whatever the new version is. I’m pretty confident that that’s all we’ll have to do because that’s all that had to be done to go from Visual Studio 2017 to 2019 and to go from 2019 to 2022. I checked the versions of [these](https://learn.microsoft.com/en-us/visualstudio/install/use-command-line-parameters-to-install-visual-studio?view=vs-2022) [three](https://learn.microsoft.com/en-us/visualstudio/install/workload-and-component-ids?view=vs-2022) [pages](https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-community?view=vs-2022) for Visual Studio 2017 and 2019, and I found that those older versions supported all of the same command-line parameters and IDs that get used here. My prediction is that the next version(s) of the Visual Studio Installer will also support those same command-line parameters and IDs. Now, Microsoft could throw us a curve ball. They could release a version of Visual Studio that breaks compatibility with those command-line parameters. Even if that happens, I doubt that it will take much effort in order to update this command. We only need to install a single workload after all.

More importantly, I think that how easy it to keep that command up to date is irrelevant. We only ever need to update that command in two different situations:

1. We switch to a new version of Visual Studio. At most, this will happen every two or three years.

2. We start depending on a component that isn’t included in the “Desktop development with C++” workload. This will probably happen infrequently or never.

Those two things don’t happen very often, so the effort required to keep the command up to date is very low. At the same time, these two commands make it easier to test the build instructions. When testing build instructions, it’s important to uninstall all build dependencies and then follow the build instructions from the beginning. If you don’t do that, then you run the risk of having build instructions that either don’t work or only work for users that have already built the software. Providing two commands that install all of the dependencies makes doing tests like that easier and less error prone. I would know. I’ve done tests like that many times while creating this PR. Having those two commands was very helpful. In other words, the small burden of keeping two commands up to date is outweighed by the benefit of being able to test the build instructions more easily and accurately.

Additionally, those two commands make it easier for new users to get started. Making things easier for users is more important than making things easier for maintainers.

---

> I'm not at all sure what the above is supposed to do.

That command makes sure that the “Desktop development with C++” workload is installed.

> Being user friendly > being fast, since what's faster for one person may well be slower for another.

I agree, being user friendly is definitely more important that being fast. That being said, I think that copy and pasting two commands is more user friendly than linking to a download page. In fact, I included those two commands in order to make the instructions more user friendly. I wasn’t thinking about speed at all when I added them.

Once those commands are started, no user interaction is required. You just have to wait for the Visual Studio Installer window to close itself. If we link to a download page, then the user has to choose a download link, wait for it to download, open the Visual Studio Installer installer, click “Continue”, wait for the Visual Studio Installer to download and install, find and check the correct workload, click “Install”, wait for it to finish, and then close the Visual Studio Installer. Copy and pasting two commands is just easier. Lgt2x said previously ‘I don't know if my explanation is clear but I feel like it's adding friction to the simple "copy 3 commands and done".’ By linking to a download page, I feel like we would be adding friction to the simple “copy 2 commands and done”.

Also, you said “what's faster for one person may well be slower for another”. Wouldn’t copy and pasting commands be the same speed for everyone?

Reply:
> Additionally, those two commands make it easier for new users to get started. Making things easier for users is more important than making things easier for maintainers.

Ok to keep the command